### PR TITLE
Group static website deploy with other deploys

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
Part of #181

File: package.json [hunk 1]
```diff
@@ -227,7 +227,7 @@
 				{
 					"command": "azureStorage.deployStaticWebsite",
 					"when": "explorerResourceIsFolder == true && config.azureStorage.preview.staticWebsites",
-					"group": "zzz_"
+					"group": "zzz_@deploy"
```
[the remark this answers is not in the document]
Comment (StephenWeatherford) [reply package.json]: That's what I was originally suggesting, but going this approach means they'll be out of sync until all three are published again.  Don't have a strong opinion, though, given the low overlap (except on PM machines :-))

[the remark this answers is not in the document]
Comment (EricJizbaMSFT) [reply package.json]: Oh... yeah I don't care about them being out of sync temporarily :)

But anyways "zzz_@deploy" seems like a weird compromise. I don't see why that's better than "zzz_@1", which is already used in the other ones and seems more aligned with the [example](https://code.visualstudio.com/docs/extensionAPI/extension-points#_sorting-inside-groups) in the docs. If you want to add "deploy" it feels like after the underscore is where that should go

**Comment by EricJizbaMSFT** [comment]
At this point this is out of date. Closing in favor of https://github.com/Microsoft/vscode-azurestorage/pull/254

**Comment by StephenWeatherford** [comment]
@EricJizbaMSFT THanks!

**Pull request closed without merging**